src/Calculator/Resources/ko-KR/Resources.resw

```diff
@@ -2907,4 +2907,12 @@
     <value>Microsoft 서비스 계약</value>
     <comment>Displayed on a link to the Microsoft Services Agreement in the about this app information</comment>
   </data>
+  <data name="UnitAbbreviation_Pyeong" xml:space="preserve">
+    <value>평</value>
+    <comment>An abbreviation for a measurement unit of area.</comment>
+  </data>
+  <data name="UnitName_Pyeong" xml:space="preserve">
+    <value>평</value>
+    <comment>A measurement unit for area.</comment>
+  </data>
```

Comment:
Unfortunately, our Localizaton system is not yet able to directly accept localization changes directly through this repo, so we should only add the english resource.

Reply:
Gotcha. I removed translations for Korean :)